### PR TITLE
Import icon

### PR DESCRIPTION
**Pull request by AnastasiiaSorina**
[Notion](https://www.notion.so/Import-icon-b06a3b64baf24e328809e101b157db5e)

- [x] create a Component icon
- [x] import all Svg to Constants. [eacc426](https://github.com/AnastasiSo/Portfolio-Project/pull/8/commits/eacc42635a812f3617a6468ef36d2b798a7c34af)

### CODE REVIEW

File: src/icon/Icon.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
+import { FunctionComponent } from "react";
```
Comment (TaniaZaslon) [inline src/icon/Icon.tsx]: Let's move Icon component under `/src/component/` folder.

**Review by TaniaZaslon — CHANGES_REQUESTED** [review]

File: src/icon/Icon.tsx (hunk 1 above)
Comment (AnastasiiaSorina) [reply src/icon/Icon.tsx]: Okay. [a85b607](https://github.com/AnastasiSo/Portfolio-Project/pull/8/commits/a85b607d204fb51b0e5d6acbec372318d7ce57ba)

Comment (TaniaZaslon) [reply src/icon/Icon.tsx]: Whole folder "icon", with constant.ts too.

Comment (AnastasiiaSorina) [reply src/icon/Icon.tsx]: Done✅ [5c2386a](https://github.com/AnastasiSo/Portfolio-Project/pull/8/commits/5c2386a2b3276c74b7b6155dd10a7131f16c396c)

**Review by TaniaZaslon — APPROVED** [review]

**Pull request merged**